media/js/ccnmtljs/bb_mixins.js

```diff
@@ -0,0 +1,40 @@
+/* Putting Mixins to use for validation in multiple views here */
+
+// A simple object with some methods
+var ValidationMixin = {
```

Comment:
These validators are cool - I haven't used mixins in plain JavaScript before (without Ember.js's classes) and it seems like a nice way to organize it.